test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/SimpleAppX86DesktopOnlyTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Server.IntegrationTesting;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.ViewCompilation
+{
+    public class SimpleAppX86DesktopOnlyTest : IClassFixture<SimpleAppX86DesktopOnlyTest.SimpleAppX86DesktopOnlyFixture>
+    {
+        public SimpleAppX86DesktopOnlyTest(SimpleAppX86DesktopOnlyFixture fixture)
+        {
+            Fixture = fixture;
+        }
+
+        public ApplicationTestFixture Fixture { get; }
+
+        [Fact(Skip = "https://github.com/dotnet/corefx/issues/20364")]
```

Comment:
Not sure why they closed that issue. Still need https://github.com/dotnet/sdk/issues/1259 to be completed before everything works as expected.